<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainwindowClass</class>
 <widget class="QMainWindow" name="MainwindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1182</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mainwindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resource.qrc">
    <normaloff>:/res/windowIcon.png</normaloff>:/res/windowIcon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton
{
	font: 25 12pt '微软雅黑';
	color: rgb(255,255,255);
	background-color: rgb(20,196,188);
	border:none;
	min-width:75px;/*控件最小宽度*/
    min-height:24px;/*控件最小高度*/
	border-radius:10px;
}
QPushButton:hover
{
	background-color: rgb(22,218,208);
}
QPushButton:pressed
{
	background-color: rgb(17,171,164);
}
#centralWidget
{
	background-color: #E7F7F3;
}
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
	font: 25 12pt '思源黑体 CN BOLD';
	color: rgb(255,255,255);
	background-color: rgb(251,102,102);
	border:none;
</string>
    </property>
    <property name="text">
     <string>我  的  日  程</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>680</y>
      <width>1200</width>
      <height>60</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>150</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="frame">
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>60</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnAdd">
          <property name="text">
           <string>添加</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnDel">
          <property name="text">
           <string>删除</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_2">
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>60</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>34</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnAddClock">
          <property name="text">
           <string>添加</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnDelClock">
          <property name="text">
           <string>删除</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>34</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>80</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-right-style: solid;
border-right-width: 1px;
border-right-color: white;
border-right: 1px solid white;

font: 25 12pt '思源黑体 CN BOLD';
color: rgb(255,255,255);
background-color: rgb(251,102,102);</string>
    </property>
    <property name="text">
     <string>时  间</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>80</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 12pt '思源黑体 CN BOLD';
color: rgb(255,255,255);
background-color: rgb(251,102,102);
border:none;</string>
    </property>
    <property name="text">
     <string>内      容</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelDate">
    <property name="geometry">
     <rect>
      <x>501</x>
      <y>110</y>
      <width>200</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 12pt '微软雅黑';
color : #F89F92;
font-weight:bold;
border:none;
background-color: #E7F7F3;</string>
    </property>
    <property name="text">
     <string>2023年6月7日 星期三</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelTime">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>170</y>
      <width>200</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Pixel LCD7</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 20pt 'Pixel LCD7';
color : #F89F92;
font-weight:bold;
border:none;
background-color: #E7F7F3;</string>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelImage">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>360</y>
      <width>281</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Resource.qrc">:/res/panda.gif</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QFrame" name="frameSettings">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>270</y>
      <width>150</width>
      <height>270</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#frameSettings
{
	background-color: #fb6666;
}
#btnSwitch
{
	min-width:0px;		/*控件最小宽度*/
	min-height:0px;	/*控件最小高度*/
	background-color: #fb6666;
	background-image: url('qrc:/res/switch_on.png');
    background-position: center;
    background-repeat: no-repeat;
}
#btnClear
{
	min-width:0px;		/*控件最小宽度*/
	min-height:0px;	/*控件最小高度*/
	background-color: #fb6666;
    background-position: center;
    background-repeat: no-repeat;
}
#btnClear:hover
{
	background-color: rgb(22,218,208);
}
#btnClear:pressed
{
	background-color: rgb(17,171,164);
}
#btnHideSettings
{
/*
	font: 25 12pt '微软雅黑';
	color: rgb(255,255,255);
	background-color: rgb(20,196,188);
	border:none;
	min-width:75px;		//控件最小宽度
    min-height:24px;	//控件最小高度
	border-radius:10px;*/
	background-color: #fb6666;
	min-width:0px;		/*控件最小宽度*/
	min-height:0px;	/*控件最小高度*/
}
#btnHideSettings:hover
{
	background-color: rgb(22,218,208);
}
QLabel
{
	font: 25 12pt '思源黑体 CN BOLD';
	color: rgb(255,255,255);
	border:border
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="btnHideSettings">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>105</y>
       <width>15</width>
       <height>80</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Resource.qrc">
       <normaloff>:/res/line_left.png</normaloff>:/res/line_left.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="btnClear">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>31</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Resource.qrc">
       <normaloff>:/res/clear.png</normaloff>:/res/clear.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSwitch">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>51</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Resource.qrc">
       <normaloff>:/res/switch_on.png</normaloff>:/res/switch_on.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>80</width>
       <height>40</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>50</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>一键清空</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>提示音</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-bottom-style: solid;
border-bottom-width: 1px;
border-bottom-color: white;
border-bottom: 1px solid white;</string>
     </property>
     <property name="text">
      <string>设  置</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnShowSettings">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>375</y>
      <width>10</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#btnShowSettings
{
/*
	font: 25 12pt '微软雅黑';
	color: rgb(255,255,255);
	background-color: rgb(20,196,188);
	border:none;*/
	border-radius:0px;
	background-color: #fb6666;
	min-width:0px;		/*控件最小宽度*/
	min-height:0px;	/*控件最小高度*/
}
#btnShowSettings:hover
{
	background-color: rgb(22,218,208);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Resource.qrc">
      <normaloff>:/res/line_right.png</normaloff>:/res/line_right.png</iconset>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
